frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -79,7 +79,8 @@ export class PieceEditor implements OnInit {
       private route: ActivatedRoute,
       private pieceAPI: PieceApi,
       private location: Location,
-      private respHandler: ResponseStatusHandlerService) {
+      private respHandler: ResponseStatusHandlerService,
+      private superDisplay: DisplayService) {
```

Comment:
This should be more specific like superBalletDisplayService or balletDisplayService

Reply:
Done.